jenkins/validate-artifacts/validate-artifacts.jenkinsfile

```diff
@@ -175,5 +225,47 @@ pipeline {
                 }
             }
         }
+        stage('Validate artifacts from urls'){
+            when {
+                expression {
+                    VERSION == '' && OPENSEARCH_ARTIFACT_URL != ''
```

Comment:
I believe this check is already done is verify-parameters stage. So can be changed to just expression `{
VERSION}` right?

Reply:
Yeah I'll check this out and update. Thanks